src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/CountOperation.java

```diff
@@ -48,11 +47,11 @@ public Uni<Supplier<CommandResult>> execute(QueryExecutor queryExecutor) {
   private SimpleStatement buildSelectQuery() {
     final List<Expression<BuiltCondition>> expressions =
         ExpressionBuilder.buildExpressions(logicalExpression, null);
-    List<Object> collect = new ArrayList<>();
-    if (expressions != null && !expressions.isEmpty() && expressions.get(0) != null) {
-      collect = ExpressionBuilder.getExpressionValuesInOrder(expressions.get(0));
-    }
-    QueryOuterClass.Query query = null;
+    //    List<Object> collect = new ArrayList<>();
```

Comment:
If these are meant to stay, need comment before to explain what it was doing; but probably should just be removed?